exporters/elasticsearch-exporter/src/test/java/io/zeebe/exporter/ElasticsearchClientTest.java

```diff
@@ -143,4 +145,22 @@ public void shouldThrowExceptionIfFailToFlushBulk() {
             "mapper_parsing_exception",
             "failed to parse");
   }
+
+  @Test
+  public void shouldIgnoreRecordIfDuplicateOfLast() {
+    // given
+    final Record<VariableRecordValue> recordMock = mock(Record.class);
+    when(recordMock.getPartitionId()).thenReturn(1);
+    when(recordMock.getValueType()).thenReturn(ValueType.WORKFLOW_INSTANCE);
+    when(recordMock.getKey()).thenReturn(RECORD_KEY + 1);
+    when(recordMock.toJson()).thenReturn("{}");
+    client.index(recordMock);
```

Comment:
A new-line could improve the reading flow.

```suggestion
    when(recordMock.toJson()).thenReturn("{}");
    
    client.index(recordMock);
```